src/main/java/it/pagopa/interop/probing/probingapi/service/impl/ProducerServiceImpl.java

```diff
@@ -14,14 +16,21 @@ public class ProducerServiceImpl implements ProducerService {
 
   @Autowired
   private Logger logger;
+
   @Autowired
   private ProducerClient producerClient;
 
   @Override
   public List<SearchProducerNameResponse> getEservicesProducers(Integer limit, Integer offset,
       String producerName) {
     logger.logMessageSearchProducer(producerName);
-    return producerClient.getProducers(limit, offset, producerName).getBody();
-  }
+    SearchProducerNameBEResponse response =
+        producerClient.getProducers(limit, offset, producerName).getBody();
 
+    return !CollectionUtils.isEmpty(response.getContent())
+        ? response.getContent().stream().map(name -> {
+          return SearchProducerNameResponse.builder().label(name).value(name).build();
+        }).toList()
+        : List.of();
+  }
```

Comment:
Am I missing something or it equivalent to this?
```suggestion
    return response.getContent().stream().map(name -> {
          return SearchProducerNameResponse.builder().label(name).value(name).build();
        }).toList();
  }
```

`response.getContent()` should never be null, right?

Reply:
Yes, `response.getContent()` should never be null, but it's a check I added for better reliability. I'll redo it if you want